modules/preparing-for-installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-install-rpm.adoc 
+
+[id="preparing-install-microshift-from-rpm-package_{context}"]
+= Preparing to install MicroShift from an RPM package
+
+Before installing MicroShift from an RPM package, you must configure your {op-system} machine to have a Logical Volume Manager (LVM) Volume Group (VG) with sufficient capacity for the Persistent Volumes (PVs) of your workload.
+
+MicroShift uses the Open Data Foundation Logical Volume Manager (ODF-LVM) Container Storage Interface (CSI) provider for storing PVs. ODF-LVM relies on Linux's LVM to dynamically manage the backing storage for PVs. For this reason, your machine must have an LVM VG in which ODF-LVM can create the LVM Logical Volumes (LVs) for your workload's PVs.  
```

Comment:
ODF-LVM may need to be renamed. Need @DanielFroehlich or @ggiguash input here.

Reply:
I believe the new name is Red Hat LVMS ("Logical Volume Manager Storage"). Using a variable (microshift-csi-plugin-name for example) would make it easy to change the value in the future.